pkgs/development/python-modules/acstore/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+  ...
+}:
+buildPythonPackage rec {
+  pname = "acstore";
+  version = "20240407";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-yubHDEZ5nwltQW8sLEAhgyaXI0svHCS3a7Mewi6cvpg=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pyyaml ];
+
+  disabled = pythonOlder "3.8";
+
+  pythonImportsCheck = [ pname ];
```

```suggestion
  pythonImportsCheck = [ "acstore" ];
```
Please don't reuse pname here, since pname and module name are semantically different.

---

Will resolve across all packages :+1: